Doc/library/asyncio-policy.rst

```diff
@@ -184,23 +186,64 @@ implementation used by the asyncio event loop:
 
       Note: loop may be ``None``.
 
+   .. method:: is_active()
+
+      Returns ``True`` is the watcher is ready to use, e.g. :py:data:`SIGCHLD`
+      handler is installed, a loop is attached etc.  Spawning a subprocess with
+      *inactive* current child watcher raises :exc:`RuntimeError`.
+
+      .. versionadded:: 3.8
+
    .. method:: close()
 
       Close the watcher.
 
       This method has to be called to ensure that underlying
       resources are cleaned-up.
 
-.. class:: SafeChildWatcher
+.. class:: ThreadedChildWatcher
+
+   This implementation starts a new waiting thread for every subprocess spawn.
+
+   The watcher can be used by asyncio code executed in non-main thread without any
+   limitation.
```

Comment:
-> "This watcher implementation works reliably even when the asyncio event loop is run in a non-main OS thread".